src/index.html

```diff
@@ -2,16 +2,154 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="get"
+      autocomplete="off"
+    >
+      <fieldset class="infBox" name="persoanlInf">
+        <legend>Personal information</legend>
+        <label class="form_inf">
+        Surname:
+          <input
+            type="text"
+            name="Surname"
+            id="one"
+          >
+        </label>
+        <label class="form_inf">
+        Name:
+          <input
+          type="text"
+          name="Name"
+          id="two"
+          >
+        </label>
+        <label class="form_inf">
+          How old are You?
+          <input
+          type="number"
+          name="Age"
+          min="1"
+          max="100"
+          value="12"
+          id="three"
+          >
```

Comment:
Go through [checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md) and fix your indents everywhere in your code, thay are not consistent
```suggestion
        <label class="form_inf">
          Name:
          <input
            type="text"
            name="Name"
            id="two"
          >
        </label>
        <label class="form_inf">
          How old are You?
          <input
            type="number"
            name="Age"
            min="1"
            max="100"
            value="12"
            id="three"
          >
```